util/weekly_build/2_GetSpackStackDevelop.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ -z $SETUPDONE ]; then . ShellSetup.sh $* ; fi
+
+set -ex
+
+cd $RUNDIR
+if [ -d $RUNID ]; then
+  cd $RUNID
+  git pull
+else
+  git clone --recurse-submodules https://github.com/AlexanderRichert-NOAA/spack-stack -b weekly_build $RUNID
```

Comment:
This needs to be configurable. On dod hpcmp machines, I need to point to the internal enterprise github (url and branch are different)

Reply:
done